chips/sam4l/src/gpio.rs

```diff
@@ -173,7 +173,7 @@ impl Port {
 
 /// Port A
 pub static mut PA: Port = Port {
-    port: unsafe { StaticRef::new((BASE_ADDRESS + 0 * SIZE) as *const GpioRegisters) },
+    port: unsafe { StaticRef::new(BASE_ADDRESS as *const GpioRegisters) },
```

Comment:
Is this a case where we want to keep this as-is for continuity with other entries?

Reply:
Well, I've come around to the "if the Rust community rallies around it, and it doesn't really matter, then better to ease the learning curve for Rust programmers learning Tock" idea, so I side with going with the correctness lints recommendations. However, it is easy to turn that one off.